src/com/itschool/session4/homework/ex1/OperatiiAritmetice.java

```diff
@@ -0,0 +1,32 @@
+package com.itschool.session4.homework.ex1;
+
+import java.util.Scanner;
+
+public class OperatiiAritmetice {
+    public static void main(String[] args) {
+
+        Scanner scanner = new Scanner(System.in);
+
+        System.out.println("Introduceti primul numar: ");
+        double nr1 = scanner.nextDouble();
+
+        System.out.println("Introduceti al doile numar: ");
+        double nr2 = scanner.nextDouble();
+
+        double adunare = nr1 + nr2;
+        double scadere = nr1 - nr2;
+        double inmultire = nr1 * nr2;
+        double impartire = nr1 / nr2;
+        double IRest = nr1 % nr2;

```

Comment:
this variable does not respect Java notation. Only classes start with uppercase letter, variables should start with lowercase letter. Please rename it.

Reply:
gata